setup.py

```diff
@@ -34,17 +34,17 @@
       setup_requires=['pytest-runner'],
       tests_require=tests_require,
       install_requires=[
-        'click>=7.0',
-        'simplejson>=3.16',
-        'pyzmq>=18.0',
-        'ruamel.yaml>=0.15',
-        'requests>=2.21',
-        'schematics>=2.1',
-        'aiohttp>=3.5',
-        'aiohttp-cors>=0.7',
+        'click==7.1.2',
+        'simplejson==3.17.6',
+        'pyzmq==22.3.0',
+        'ruamel.yaml==0.17.17',
+        'requests==2.26.0',
+        'schematics==2.1.1',
+        'aiohttp==3.7.4',
+        'aiohttp-cors==0.7.0',
         'emloop>=0.2',
-        'apistrap>=0.9.1',
-        'minio>=4.0',
+        'apistrap==0.9.11',
+        'minio==5.0.6',
```

Comment:
Is it really necessary to be this specific? Not that it matters much...

Reply:
@teyras well, kinda yes. It downloaded minio 7 something and didn't work. So i took packages that I already proven in isletnet as working together and specified them hereb as well.